lib/tasks/stimulus_reflex/install.rake

```diff
@@ -57,6 +57,20 @@ namespace :stimulus_reflex do
       File.open(filepath, "w") { |f| f.write lines.join }
     end
 
+    lines = File.open(filepath, "r") { |f| f.readlines }
```

Comment:
Is there a specific reason we call `File.open(filepath, "r") { |f| f.readlines }` all over the install script? 
It seems that `File.readlines(filepath)` is doing the same

```suggestion
    lines = File.readlines(filepath)
```

Reply:
![cargo-cult-plane_497x375](https://user-images.githubusercontent.com/38150464/119280502-c02a8a00-bbff-11eb-9911-b5f9731328cb.jpg)